src/sentry/features/manager.py

```diff
@@ -178,6 +181,9 @@ def add(
             feature_option_name = f"{FLAGPOLE_OPTION_PREFIX}.{name}"
             register(feature_option_name, type=Dict, default={}, flags=FLAG_AUTOMATOR_MODIFIABLE)
 
+        if default is not None:
+            settings.SENTRY_FEATURES[name] = default
```

Comment:
I think this should be `if default is not None and name not in settings.SENTRY_FEATURES`. This would allow customers to still override feature flag settings via settings, which I think they do today?